src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs

```diff
@@ -18,6 +18,12 @@ namespace System.Net.WebSockets.Client.Tests
 {
     public partial class ClientWebSocketOptionsTests : ClientWebSocketTestBase
     {
+        // Recreate the minimum WebSocket buffer size values from the Framework version of WebSocket.
+        // We need these values to test when the minimums are violated, but the values set in Framework
+        // are internal to the WebSocketBuffer class.
+        private const int MinSendBufferSize = 16;
+        private const int MinReceiveBufferSize = 256;
+
```

Comment:
There is a lot of duplicate code below.  And the only difference is the value being passed to SetBuffer.

Instead of using const int here, why not just use a variable and set it appropriately.  Then the rest of the code can be much simpler and less duplication, i.e.

```c#
int minSendBufferSize = PlatformDetection.IsFullFramework ? 16 : 1;
int minReceiveBufferSize = PlatformDetection.IsFullFramework ? 256 : 1;
```

And in other places below, the code is identical in the if and else clauses:

```c#
AssertExtensions.Throws<ArgumentOutOfRangeException>("receiveBufferSize", () => cws.Options.SetBuffer(0, 0, new ArraySegment<byte>(new byte[1])));
```

And these variables don't even have to be class level.  It seems like they only need to be in the test themselves as local variables.